webpack.config.js

```diff
@@ -34,14 +34,22 @@ module.exports = draftConfig => {
     },
 
     plugins: [
+      new webpack.EnvironmentPlugin({
+        NODE_ENV: 'development', // use 'development' unless process.env.NODE_ENV is defined
```

Comment:
>use 'development' unless process.env.NODE_ENV is defined

Wouldn't this do that?
```suggestion
        NODE_ENV: process.env.NODE_ENV || 'development',
```